tools/pl-paths.js

```diff
@@ -20,11 +20,20 @@ const {
 //   // ...
 // }
 
+// all particle paths
 const paths = [].concat(
   ...Object.values(patternPaths).map(atomic => Object.values(atomic))
 );
 
+// only check our actual components from atoms up
+delete patternPaths.protons;
+delete patternPaths.atoms.svgicons;
+const componentPaths = [].concat(
+  ...Object.values(patternPaths).map(atomic => Object.values(atomic))
```

Comment:
Check this thing out.

Reply:
hah, it's essentially a carbon copy of the "paths" variable above, just with some other things i don't necessarily care about for accessibility testing removed. original paths is still available for VRT testing